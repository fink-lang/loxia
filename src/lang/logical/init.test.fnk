{fink2js} = import '../../testing/generate.fnk'
{describe, it, expect, to_match_snapshot} = import '@fink/jest/test.fnk'


describe 'logical', fn:
  it 'compiles', fn:
    expect
      fink2js "
        foo = a or b or c
        bar = a and b and c
        spam = not a
      "
      to_match_snapshot
