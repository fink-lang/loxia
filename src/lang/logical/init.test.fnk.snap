// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`logical compiles 1`] = `
"export const foo = a || b || c;
export const bar = a && b && c;
export const spam = !a;"
`;
